### ISSUE TITLE
OTS parsing error: cmap: Failed to parse table

### ISSUE THREAD

**Issue by ackzell** [lusaxweb/iconsax#5]
Hi there!

This is a very sweet set of icons, thanks for sharing!

I have one issue though: when I try to use the provided downloads (the CSS and font files) on a project, I get the following errors on my browser:
```
Failed to decode downloaded font: http://127.0.0.1:8080/fonts/iconsax.ttf?yjegnn
127.0.0.1/:1 OTS parsing error: cmap: Failed to parse table
127.0.0.1/:1 Failed to decode downloaded font: http://127.0.0.1:8080/fonts/iconsax.woff?yjegnn
127.0.0.1/:1 OTS parsing error: cmap: Failed to parse table
``` 

So I went ahead and used an SVG to build my own font via the [IcoMoon App](https://icomoon.io/app/#/select).
**Note**: I actually needed to convert strokes to paths in order for it to work, so I was only able to create a font file with 1 icon.

At the end I was able to load the font properly, but obviously this is only 1 icon, so I can't use it on my project. 

Would you be able to confirm this is an issue and provide a new version of the font files?

Thanks again! 

## TL;DR:

Using the download from iconsax website:
![image](https://user-images.githubusercontent.com/601266/131595582-90c15fc2-ea9a-4806-9f2f-c1d5ad5349f2.png)

After creating my own font with IcoMoon:
![image](https://user-images.githubusercontent.com/601266/131591183-5f2cabfc-1022-40b9-8509-cb13ffdf19cf.png)

**Comment by ilyasbilgihan** [comment]
I have had the same issue and I fixed it by using [Transfonter](https://transfonter.org/). 

Go to Transfonter > click `Add fonts` button > choose the `ttf` file you downloaded from this repo > check `ttf`, `woff`, `svg`, `eot` select boxes from `formats` section below. Then click green `Convert` button. It will give you a zip file which includes font files with the extensions we selected before. Replace the downloaded font files from this repo with the new font files from Transfonter.

**Comment by ackzell** [comment]
Thank you @ilyasbilgihan ! it worked really well!

**Comment by ackzell** [comment]
Closing the issue, but maybe worth re-uploading the files to https://iconsax.io/ @luisDanielRoviraContreras ?

**Comment by ilyasbilgihan** [comment]
> Closing the issue, but maybe worth re-uploading the files to https://iconsax.io/ @luisDanielRoviraContreras ?

**update:** For those who want to use updated font files, please take a look at PR #23

**Comment by floranknz** [comment]
We are 3 years from the original bug and the fonts on https://iconsax.io/ are still the old wrong ones. Might be worth updating them on the website again @luisDanielRoviraContreras ?
